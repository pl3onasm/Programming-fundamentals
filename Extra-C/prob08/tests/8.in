1500:[997,1004),[93,96),[491,492),[1431,1432),[1059,1066),[1174,1177),[268,274),[497,504),[4,5),[1212,1214),[166,169),[1405,1406),[1302,1310),[1042,1043),[245,248),[1141,1145),[574,582),[388,396),[1177,1183),[146,152),[65,66),[145,151),[365,373),[499,502),[262,265),[737,744),[328,333),[351,357),[819,824),[112,118),[885,889),[1136,1140),[1265,1270),[751,752),[579,583),[341,347),[1050,1057),[564,571),[678,679),[795,797),[23,24),[1381,1386),[378,383),[766,773),[1116,1119),[1185,1188),[235,236),[74,80),[371,375),[166,167),[449,453),[689,692),[1322,1324),[839,847),[517,519),[956,964),[40,45),[454,460),[907,911),[54,56),[1019,1025),[1004,1010),[7,13),[869,877),[866,871),[832,835),[1276,1283),[921,929),[576,579),[817,821),[426,431),[294,295),[1137,1145),[387,394),[275,282),[705,706),[387,393),[613,614),[1361,1368),[475,482),[800,804),[32,33),[486,494),[76,81),[556,560),[174,179),[1476,1478),[1119,1121),[1327,1331),[178,186),[769,770),[745,746),[567,575),[589,590),[60,68),[1055,1058),[770,778),[613,619),[1193,1201),[87,90),[507,508),[1311,1315),[1041,1049),[509,511),[1184,1192),[1123,1124),[247,252),[438,442),[741,747),[1104,1105),[250,253),[675,676),[31,37),[102,110),[902,904),[1009,1011),[727,733),[280,287),[1211,1216),[354,360),[519,520),[69,74),[1,5),[222,228),[491,493),[944,949),[1256,1261),[877,884),[946,949),[1130,1135),[550,551),[1438,1443),[975,978),[1228,1235),[35,39),[989,995),[873,881),[1454,1458),[349,357),[1398,1401),[382,390),[463,465),[477,482),[273,279),[1363,1367),[233,241),[429,435),[1274,1281),[638,643),[1081,1087),[929,931),[687,695),[142,150),[535,543),[1265,1270),[811,812),[202,204),[1184,1186),[105,108),[188,193),[1495,1496),[594,596),[1453,1459),[1143,1148),[1101,1104),[1097,1098),[453,458),[88,96),[877,885),[428,433),[345,347),[1250,1258),[1383,1390),[288,296),[1045,1049),[582,585),[457,464),[1364,1369),[105,113),[1288,1292),[148,155),[727,728),[105,111),[989,990),[439,445),[21,28),[357,362),[57,60),[578,585),[1024,1026),[465,469),[729,734),[679,687),[185,192),[340,348),[1022,1027),[1133,1137),[1480,1486),[28,33),[986,994),[1356,1359),[126,131),[490,491),[468,472),[1220,1228),[1406,1407),[738,741),[1456,1458),[935,937),[970,976),[164,167),[237,245),[376,383),[1498,1499),[912,914),[1207,1211),[657,659),[201,209),[177,182),[469,473),[970,971),[444,452),[429,437),[605,607),[625,632),[302,308),[721,725),[581,587),[189,193),[781,782),[39,47),[1476,1482),[1143,1144),[566,572),[1485,1489),[1467,1472),[937,944),[1441,1447),[939,944),[1341,1348),[679,681),[1026,1030),[714,718),[510,513),[1218,1225),[379,380),[1444,1449),[489,493),[1172,1174),[534,539),[1107,1111),[322,327),[1093,1098),[424,431),[530,533),[1458,1465),[1444,1449),[462,467),[958,966),[993,995),[1188,1193),[54,57),[630,638),[1439,1441),[649,651),[79,83),[662,665),[719,724),[536,544),[552,558),[77,80),[531,532),[518,520),[1142,1143),[341,342),[240,243),[796,801),[1096,1097),[896,901),[1239,1240),[894,896),[129,137),[1487,1495),[1316,1318),[280,286),[546,554),[1408,1409),[1155,1156),[1126,1127),[642,644),[96,102),[319,324),[1337,1339),[669,674),[1386,1391),[412,415),[869,871),[173,181),[1499,1502),[801,804),[172,177),[1380,1387),[1238,1240),[1235,1238),[692,697),[978,986),[1001,1005),[152,160),[212,216),[1238,1241),[654,661),[407,411),[770,772),[1094,1102),[883,890),[920,922),[901,903),[1175,1183),[118,126),[338,343),[377,381),[168,174),[1312,1317),[899,907),[1203,1204),[154,161),[61,68),[1051,1056),[770,775),[1462,1470),[179,180),[94,102),[619,625),[1343,1344),[103,108),[1318,1326),[408,415),[207,210),[1099,1106),[767,774),[1124,1130),[202,208),[613,616),[1072,1080),[173,175),[880,888),[1315,1319),[1023,1029),[713,715),[1172,1177),[805,812),[1214,1218),[1319,1324),[965,968),[67,69),[1195,1198),[342,344),[915,923),[1058,1065),[410,411),[907,912),[852,855),[648,652),[61,62),[135,140),[9,14),[654,658),[1129,1131),[37,41),[58,59),[929,930),[673,675),[878,884),[1450,1457),[710,712),[1469,1474),[294,302),[734,739),[1121,1127),[810,813),[1248,1254),[1459,1465),[1391,1399),[877,885),[857,864),[1147,1154),[949,956),[530,532),[922,927),[573,579),[147,154),[139,145),[1313,1316),[1242,1250),[667,672),[1148,1153),[1162,1167),[643,651),[1390,1394),[345,349),[728,732),[628,631),[338,344),[723,724),[248,252),[1413,1418),[1119,1122),[1253,1259),[1081,1086),[73,77),[1199,1205),[354,357),[1039,1044),[453,456),[17,22),[272,274),[1416,1421),[957,958),[1215,1216),[71,72),[654,659),[107,114),[1050,1052),[579,584),[765,768),[1416,1422),[569,575),[821,825),[1494,1496),[196,197),[249,252),[355,362),[54,55),[154,160),[147,154),[1003,1009),[317,318),[950,954),[689,690),[1057,1063),[628,631),[1137,1143),[655,657),[1276,1283),[1157,1158),[1455,1456),[1021,1024),[448,451),[897,903),[450,453),[273,276),[416,418),[1426,1429),[87,89),[1158,1159),[1098,1102),[279,280),[219,222),[944,950),[1072,1079),[545,552),[1249,1252),[579,586),[1262,1266),[703,711),[0,7),[1144,1146),[1313,1316),[190,195),[631,637),[456,463),[131,137),[824,826),[1364,1367),[104,109),[1235,1238),[1172,1180),[928,933),[780,781),[981,987),[660,663),[778,783),[786,787),[110,114),[818,823),[901,903),[221,227),[929,935),[177,179),[460,461),[358,366),[951,958),[608,616),[382,388),[37,41),[1268,1273),[1218,1220),[850,855),[855,859),[812,817),[925,933),[337,341),[260,262),[213,217),[1086,1087),[491,494),[839,847),[1355,1360),[1387,1395),[870,878),[729,734),[136,139),[1014,1017),[1091,1095),[1494,1496),[1373,1376),[345,346),[563,567),[1054,1062),[529,531),[708,709),[1402,1403),[1339,1347),[689,693),[303,309),[1128,1131),[1200,1202),[694,696),[557,563),[568,574),[1363,1370),[562,565),[1371,1375),[482,487),[852,856),[1035,1043),[922,924),[814,819),[1030,1032),[732,740),[120,126),[404,408),[832,840),[1024,1025),[890,891),[1421,1428),[1463,1468),[987,995),[1479,1483),[263,269),[1384,1388),[1411,1414),[240,247),[633,639),[122,127),[1322,1329),[1317,1321),[29,33),[1040,1048),[311,315),[511,512),[725,732),[1433,1439),[1255,1258),[1178,1184),[502,504),[609,615),[1327,1335),[522,523),[1283,1285),[818,825),[488,494),[141,145),[491,499),[14,18),[428,432),[1402,1404),[392,394),[815,816),[1498,1503),[1212,1213),[740,743),[414,418),[1419,1427),[1099,1103),[709,714),[805,810),[1280,1283),[456,459),[618,622),[1258,1262),[733,735),[588,592),[629,630),[650,658),[967,972),[154,156),[1430,1435),[260,266),[572,575),[460,467),[667,669),[1351,1353),[1159,1164),[672,673),[12,20),[252,259),[1063,1067),[1275,1281),[103,111),[536,543),[629,630),[192,199),[401,409),[1277,1281),[1260,1266),[14,20),[931,936),[1026,1028),[441,443),[98,104),[737,745),[52,53),[829,835),[107,114),[152,154),[532,537),[921,929),[773,777),[1166,1167),[922,925),[1047,1055),[1352,1358),[1295,1296),[1244,1248),[233,236),[264,266),[1332,1336),[82,84),[1370,1378),[431,436),[515,517),[370,372),[743,750),[1008,1011),[415,419),[837,845),[789,790),[1487,1489),[222,225),[1030,1034),[569,571),[885,891),[940,946),[1431,1435),[265,273),[1121,1129),[134,142),[70,72),[1499,1500),[1428,1431),[1203,1204),[197,200),[1193,1195),[31,32),[480,482),[1132,1139),[724,730),[577,584),[765,771),[1456,1459),[1374,1380),[331,335),[429,434),[407,412),[996,1004),[498,504),[1261,1264),[325,332),[566,568),[386,393),[756,761),[211,213),[512,516),[643,649),[1011,1015),[1350,1355),[210,214),[960,966),[39,43),[344,345),[871,874),[224,229),[1277,1283),[1001,1006),[1191,1196),[918,922),[660,667),[856,863),[822,828),[1416,1424),[23,24),[168,175),[657,663),[664,668),[225,230),[1244,1252),[201,206),[1446,1447),[789,797),[1074,1076),[32,37),[241,246),[667,669),[1211,1213),[933,941),[975,979),[1254,1262),[245,252),[1008,1015),[136,141),[740,747),[663,668),[154,157),[846,850),[61,64),[1498,1505),[903,905),[148,153),[434,437),[878,882),[1156,1161),[234,241),[59,64),[836,838),[1208,1215),[1315,1318),[563,569),[865,869),[469,473),[626,633),[1385,1388),[775,782),[1264,1272),[246,252),[590,593),[19,22),[207,213),[1326,1330),[1227,1229),[554,556),[936,939),[1039,1042),[229,236),[1145,1148),[515,520),[652,658),[163,169),[827,830),[576,583),[1235,1241),[729,730),[191,199),[339,341),[1374,1375),[486,493),[359,361),[535,539),[784,787),[270,271),[470,475),[1455,1459),[790,795),[208,213),[1391,1393),[1177,1185),[1445,1453),[416,418),[187,189),[786,791),[51,52),[1363,1368),[1324,1327),[911,919),[466,469),[1490,1492),[816,817),[599,600),[676,680),[716,718),[433,434),[1384,1389),[882,888),[785,791),[1104,1110),[370,372),[606,607),[813,819),[1304,1307),[823,825),[1142,1150),[395,396),[695,703),[209,211),[1237,1238),[801,808),[775,781),[996,997),[1194,1196),[1312,1313),[404,406),[1281,1282),[184,192),[607,611),[980,981),[205,210),[134,141),[758,759),[1194,1196),[704,710),[90,97),[1114,1120),[1000,1006),[826,828),[899,900),[1148,1152),[244,246),[1122,1128),[156,163),[1079,1083),[536,537),[311,317),[1124,1131),[480,485),[113,120),[600,607),[954,960),[332,336),[304,306),[95,97),[577,583),[672,677),[909,915),[1336,1344),[676,678),[652,659),[105,109),[1208,1209),[642,645),[221,225),[789,791),[513,521),[1180,1188),[177,184),[1008,1009),[1234,1242),[929,932),[1149,1156),[551,552),[284,290),[369,372),[958,961),[1067,1068),[612,619),[275,280),[306,309),[654,655),[727,735),[338,341),[572,576),[289,294),[1167,1174),[1440,1448),[1221,1222),[855,862),[889,896),[745,746),[1217,1224),[1386,1387),[38,43),[1100,1104),[1296,1300),[982,989),[1202,1206),[201,202),[22,26),[956,959),[1231,1232),[138,140),[56,64),[1222,1227),[51,54),[671,678),[615,616),[1455,1460),[872,873),[47,53),[81,86),[1338,1345),[1037,1042),[1440,1441),[846,850),[268,273),[1270,1277),[399,406),[340,342),[1284,1292),[246,251),[212,218),[79,85),[264,265),[1160,1168),[1457,1463),[190,192),[876,877),[589,595),[1050,1056),[242,248),[670,674),[70,78),[232,238),[879,887),[312,319),[350,357),[112,116),[331,334),[201,204),[1008,1015),[117,123),[294,297),[780,787),[426,434),[1266,1268),[823,827),[1053,1055),[1162,1163),[198,206),[1428,1431),[1068,1076),[250,253),[949,957),[1179,1180),[1388,1393),[69,77),[141,146),[349,356),[107,108),[1131,1138),[1301,1307),[138,141),[552,553),[1281,1288),[751,759),[49,52),[853,858),[444,447),[682,688),[1079,1083),[188,194),[575,578),[68,73),[1405,1406),[778,779),[1489,1491),[1166,1168),[473,480),[1484,1489),[710,715),[306,311),[933,936),[1227,1228),[541,545),[87,89),[997,1005),[1393,1394),[818,820),[787,794),[373,376),[620,623),[155,157),[626,631),[162,169),[1447,1454),[1384,1391),[90,97),[1161,1163),[1104,1112),[633,637),[375,381),[1180,1181),[935,938),[101,108),[65,68),[973,979),[1269,1277),[1208,1216),[1071,1078),[922,929),[56,62),[1349,1351),[1292,1297),[299,302),[435,440),[1486,1488),[1189,1197),[1186,1192),[1187,1193),[957,958),[85,90),[631,634),[30,38),[983,990),[346,351),[569,576),[386,391),[1276,1284),[1394,1397),[1120,1122),[616,619),[562,564),[48,51),[210,216),[257,260),[340,342),[1060,1067),[106,109),[836,841),[918,921),[38,43),[685,687),[152,156),[1413,1421),[140,148),[1404,1412),[1330,1331),[1081,1089),[635,642),[1486,1488),[2,10),[237,244),[477,478),[1194,1195),[570,575),[292,294),[536,539),[1295,1299),[7,8),[1120,1122),[47,52),[368,372),[1171,1177),[1437,1443),[613,620),[308,314),[1348,1349),[657,664),[343,345),[541,546),[1003,1006),[604,606),[964,966),[1195,1202),[231,234),[454,456),[8,14),[589,594),[527,535),[433,439),[964,970),[8,11),[1025,1026),[430,437),[22,27),[474,481),[196,201),[803,808),[202,207),[1049,1052),[1147,1155),[69,75),[154,160),[28,31),[680,686),[582,589),[1042,1043),[883,889),[236,241),[885,893),[340,346),[130,133),[864,871),[803,810),[236,237),[1202,1210),[1359,1362),[1168,1174),[469,476),[341,345),[1384,1385),[1252,1258),[1117,1118),[198,199),[1052,1053),[942,947),[117,120),[590,594),[333,335),[34,42),[401,404),[637,644),[353,360),[104,111),[956,958),[1263,1269),[119,121),[595,598),[405,413),[483,491),[571,576),[1293,1300),[140,145),[1074,1082),[1360,1362),[1214,1220),[1309,1312),[1211,1213),[1271,1276),[387,392),[577,578),[922,930),[60,68),[737,739),[913,914),[1289,1296),[242,246),[218,225),[556,561),[199,202),[49,52),[334,339),[817,823),[314,315),[1069,1073),[1475,1481),[623,629),[46,48),[478,482),[1430,1435),[150,151),[445,448),[503,506),[1271,1275),[103,109),[1022,1029),[114,119),[186,192),[561,562),[309,316),[905,909),[1039,1045),[1071,1078),[259,263),[934,938),[17,19),[479,485),[1101,1104),[735,741),[1110,1112),[117,123),[1152,1159),[185,186),[1430,1438),[476,478),[332,339),[1209,1210),[135,141),[991,999),[461,467),[973,980),[503,510),[240,243),[1390,1393),[67,68),[1213,1216),[1281,1288),[90,98),[1008,1014),[525,532),[1080,1083),[1197,1198),[1466,1469),[1340,1345),[828,836),[1408,1413),[637,643),[1097,1104),[1034,1035),[1377,1381),[1436,1443),[16,24),[704,707),[904,912),[172,177),[672,679),[326,333),[1154,1160),[1312,1313),[528,530),[1115,1120),[744,749),[56,60),[1163,1165),[1182,1190),[1420,1425),[567,572),[205,208),[343,348),[791,796),[588,596),[121,124),[1150,1157),[115,117),[826,833),[1115,1123),[79,81),[147,149),[62,64),[171,175),[1025,1033),[244,252),[573,581),[1440,1444),[725,726),[29,33),[452,458),[378,382),[677,678),[1085,1089),[204,208),[1027,1028),[383,384),[562,566),[1274,1279),[1395,1398),[1005,1007),[1419,1424),[188,194),[1190,1197),[603,605),[364,371),[486,494),[286,287),[175,176),[1431,1434),[362,368),[1143,1151),[1486,1491),[1053,1057),[68,69),[360,364),[1451,1452),[882,883),[1382,1389),[305,306),[1208,1212),[1420,1424),[76,80),[462,464),[459,460),[74,76),[1024,1030),[12,20),[214,218),[908,910),[975,983),[246,248),[916,920),[436,440),[1395,1400),[612,615),[1186,1193),[1223,1228),[1254,1261),[1460,1467),[1403,1408),[355,362),[18,25),[1239,1244),[1260,1267),[568,574),[786,790),[1397,1401),[943,947),[362,369),[927,931),[1028,1031),[1297,1305),[255,262),[417,418),[1441,1446),[179,185),[836,839),[1178,1180),[442,446),[518,521),[200,202),[661,669),[773,776),[559,565),[1373,1380),[202,207),[914,920),[464,472),[1499,1504),[742,747),[1020,1022),[1283,1289),[882,886),[1319,1326),[1438,1439),[588,594),[279,283),[279,282),[650,655),[695,700),[591,597),[305,310),[1338,1345),[1286,1294),[111,116),[785,790),[949,957),[1481,1483),[1390,1395),[1076,1082),[612,615),[879,885),[1151,1157),[280,284),[1339,1347),[282,286),[539,545),[353,356),[730,735),[1309,1313),[1405,1413),[960,965),[1474,1475),[1464,1471),[914,917),[577,581),[1266,1270),[1451,1452),[527,532),[154,160),[186,188),[792,797),[1454,1460),[548,555),[543,549),[1292,1297),[984,985),[941,948),[1487,1491),[1194,1196),[1213,1215),[863,868),[1455,1463),[199,205),[89,96),[387,388),[1126,1134),[223,230),[740,744),[397,405),[411,417),[1178,1186),[282,290),[1205,1206),[646,650),[547,548),[240,247),[316,319),[314,316),[512,518),[86,87),[984,986),[1496,1500),[1490,1498),[205,213),[924,926),[977,980),[603,608),[1220,1223),[210,218),[670,675),[1237,1242),[1353,1358),[617,619),[598,600),[1436,1437),[1034,1036),[1039,1041),[266,274),[488,495),[635,636),[927,933),[16,20),[1226,1227),[1169,1170),[1056,1060),[362,370),[832,834),[142,147),[1280,1285),[221,229),[442,443),[1317,1322),[346,347),[290,294),[1209,1215),[827,834),[1074,1080),[1391,1398),[866,872),[454,460),[713,721),[518,523),[686,693),[791,798),[730,735),[1426,1433),[963,971),[1414,1420),[395,400),[246,250),[514,516),[763,767),[1295,1296),[281,289),[920,923),[1468,1473),[1497,1501),[580,581),[972,973),[869,870),[959,963),[1010,1018),[331,335),[389,391),[349,355),[316,322),[69,70),[441,447),[1135,1141),[569,572),[124,127),[479,481),[112,120),[1460,1466),[650,656),[0,7),[68,72),[1224,1231),[893,896),[572,575),[1195,1199),[617,621),[1032,1035),[1083,1088),[665,667),[793,796),[1130,1137),[755,760),[586,591),[1098,1100),[1234,1242),[739,747),[307,313),[1264,1267),[1210,1216),[1378,1381),[519,522),[153,155),[1085,1087),[867,868),[249,251),[392,398),[351,357),[596,599),[956,957),[113,115),[253,261),[1429,1437),[721,728),[305,313),[917,922),[947,955),[674,678),[1402,1409)
