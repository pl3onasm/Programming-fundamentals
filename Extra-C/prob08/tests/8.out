[0,419),[424,523),[525,750),[751,761),[763,1018),[1019,1089),[1091,1112),[1114,1401),[1402,1505)
